core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -251,6 +251,12 @@ public void setCivicUrl(String property) {
             }
         }
     }
+    
+    public static final String USE_CUSTOM_DRIVER_ANNOTATIONS = "oncoprint.custom_driver_annotation.default";
+    public static final String BINARY_CUSTOM_DRIVER_ANNOTATION_MENU_LABEL = "oncoprint.custom_driver_annotation.binary.menu_label";
+    public static final String CLASS_CUSTOM_DRIVER_ANNOTATION_MENU_LABEL = "oncoprint.custom_driver_annotation.class.menu_label";
+    public static final String SHOW_BINARY_CUSTOM_DRIVER_ANNOTATION = "oncoprint.custom_driver_annotation.binary.show";
```

Comment:
hm...not sure it this makes sense. One will probably be looking to change these labels anyway, so there is little added value in having a default value for them. I prefer the added value of 2 less portal properties.